client/reader/index.js

```diff
@@ -63,7 +63,14 @@ module.exports = function() {
 		page( '/tag/:tag', updateLastRoute, controller.removePost, controller.sidebar, controller.tagListing );
 	}
 
-	page( '/read/a8c', updateLastRoute, controller.removePost, controller.sidebar, forceTeamA8C, controller.readA8C );
+	if ( config.isEnabled( 'reader/a8c' ) ) {
+		page( '/read/a8c', updateLastRoute, controller.removePost, controller.sidebar, forceTeamA8C, controller.readA8C );
+	}
+
+	if ( config.isEnabled( 'reader/warmstart' ) ) {
```

Comment:
Thanks @mtias! I think separating this out is a great idea. Done and done!

Separated out the feature check of /read/a8c: https://github.com/Automattic/wp-calypso/pull/5188